### PR TITLE
Add child properties for ToolItemGroup

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
Part of #392.

I think the code isn't very good. I intend to create a macro to avoid code repetition but before that, I need to be sure that the code is the one expected.

cc @gkoz @EPashkin

### CODE REVIEW

File: src/tool_item_group.rs [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    pub fn set_item_expand<T: IsA<ToolItem> + IsA<Object> + IsA<Widget>>(&self, item: &T, expand: bool) {
+        let object = self.clone();
+        unsafe { ffi::gtk_container_child_set_property(object.upcast::<Container>().to_glib_none().0,
+                                                       item.to_glib_none().0,
+                                                       "expand".to_glib_none().0,
+                                                       Value::from(&expand).to_glib_none().0) }
+    }
+
+    pub fn get_item_expand<T: IsA<ToolItem> + IsA<Object>+ IsA<Widget>>(&self, item: &T) -> bool {
+        let mut value = unsafe { Value::uninitialized() };
```
Comment (EPashkin) [inline src/tool_item_group.rs]: For `i32` this not work, IMHO for `bool` too.
I use `let mut value = 0.to_value();`

File: src/tool_item_group.rs [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+use IsA;
+use glib::translate::*;
+use glib::Value;
+use Container;
+use Widget;
+
+impl ToolItemGroup {
+    pub fn set_item_expand<T: IsA<ToolItem> + IsA<Object> + IsA<Widget>>(&self, item: &T, expand: bool) {
+        let object = self.clone();
+        unsafe { ffi::gtk_container_child_set_property(object.upcast::<Container>().to_glib_none().0,
```
Comment (EPashkin) [inline src/tool_item_group.rs]: For grid it works fine without `object`

File: src/tool_item_group.rs (hunk 1 above)
Comment (GuillaumeGomez) [inline src/tool_item_group.rs]: Great idea!

**Comment by GuillaumeGomez** [comment]
Thanks to @EPashkin, it's way better now!

**Comment by GuillaumeGomez** [comment]
@EPashkin: So, what do we do about this?

**Comment by EPashkin** [comment]
@GuillaumeGomez we don't have @gkoz's opinion, maybe he have better solution.
If we use local macros (as in #395) I like to hear your option: there better current multiproperty macro or one macro usage per property? Now I think that last option better.
Also we don't have tested code for string properties.

**Comment by GuillaumeGomez** [comment]
Like I said, my primary concern about the macro usage is what the generated docs will look like. I need to test first.

**Comment by GuillaumeGomez** [comment]
Yeay! \o/

**Comment by GuillaumeGomez** [comment]
All ok. Do you want to add something or can I merge?

**Comment by EPashkin** [comment]
All ok, thanks

**Comment by GuillaumeGomez** [comment]
Great, thanks for this! :)

**Pull request merged**